app/models/miq_report_result.rb

```diff
@@ -286,6 +286,31 @@ def get_generated_result(result_type)
     result_type.to_sym == :html ? html_rows : report_results
   end
 
+  # Get the status for the MiqReportResult via two methods:
+  #
+  # * First:  try the task_status column.  This is set when queried using the
+  #   with_task_status class method
+  # * Second: fallback to using the status method, which will do a lookup on
+  #   the MiqTask if it isn't already associated with the current instance
+  def get_status
```

Comment:
Yeah, I am in the same boat.  My rational with using a java style `get_` prefix (besides not wanting to overwrite the `status` method further) was that it emphasizes the fact that there is potentially a database call involved when doing this command.  Not sure that racking my brain to get a more terse/correct name is worth it though.